packages/vite/src/client/client.ts

```diff
@@ -364,6 +364,7 @@ export function updateStyle(id: string, content: string): void {
     if (!style) {
       style = document.createElement('style')
       style.setAttribute('type', 'text/css')
+      style.setAttribute('data-id', id)
```

Comment:
I like this idea! Maybe we should use `data-vite-id` just to be safe?

```suggestion
      style.setAttribute('data-vite-id', id)
```